S17-procasync/basic.t

```diff
@@ -106,3 +106,12 @@ throws-like { Proc::Async.new }, X::Multi::NoMatch,
     is $captured, "Hello World\n",
         "Tapping stdout supply after start of process does not lose data";
 }
+
+$pc = Proc::Async.new("perl6", "-e ''"); # It taps a warning, but it is safe to catch.
```

Comment:
Would you change this to `$*EXECUTABLE` instead of `"perl6"`?

That way we don't rely on there being an executable named perl6.